tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1798,6 +1798,86 @@ def func(self, xp, a):
 
 
 @op_utils.op_test(['native:0', 'cuda:0'])
+@pytest.mark.parametrize('lhs, rhs', [
+    (numpy.asarray([[1., 2.], [4., 5.]]), numpy.asarray([1., 3.])),
+    (numpy.asarray([1., 2., 3.]), numpy.asarray([3., 4., 5.]))
+])
+@pytest.mark.parametrize('contiguous', [None, 'C'])
+class TestPow(op_utils.NumpyOpTest):
```

Comment:
As for the tests in this PR, could you parameterize them similarly to e.g. https://github.com/chainer/chainer/blob/137224ebbd92298e6a6c58e79b7c6886217d51df/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L1373-L1409 to covert shape and value edge cases as well?